deepdreamer/deepdreamer.py

```diff
@@ -148,14 +148,19 @@ def deepdream(
         img_path, zoom=True, scale_coefficient=0.05, irange=100, iter_n=10,
         octave_n=4, octave_scale=1.4, end="inception_4c/output", clip=True,
         network="bvlc_googlenet", gif=False, reverse=False, duration=0.1,
-        loop=False):
+        loop=False, gpu=False, gpuid=0):
     img = np.float32(img_open(img_path))
     s = scale_coefficient
     h, w = img.shape[:2]
 
+    if gpu:
+        print("Enabling GPU {}...".format(gpuid))
+        caffe.set_device(gpuid)
```

Comment:
`gpuid` is passed to `deepdream` as a parameter, and it is set to `gpuid=args.gpuid` in `deepdreamer.py`.